make/wag.mk

```diff
@@ -1,23 +1,34 @@
 # This is the default Clever Wag Makefile.
 # Please do not alter this file directly.
-WAG_MK_VERSION := 0.1.0
+WAG_MK_VERSION := 0.2.0
 SYSTEM := $(shell uname -a | cut -d" " -f1 | tr '[:upper:]' '[:lower:]')
+WAG_INSTALLED := $(shell [[ -e "bin/wag" ]] && bin/wag --version)
+WAG_LATEST := $(shell curl -S https://api.github.com/repos/Clever/wag/releases/latest 2>/dev/null | grep tag_name | cut -d\" -f4)
```

Comment:
Also I actually would advocate for switching this back to `=` from `:=` since otherwise you'll be hitting loading the latest version from github every time someone runs a make target (even a non-wag related one, like building their code). This wouldn't be a huge deal, but the GH API does have rate limits (I ran into them earlier today).